virttest/libvirt_vm.py

```diff
@@ -2253,7 +2254,10 @@ def destroy(self, gracefully=True, free_mac_addresses=True):
 
     def remove(self):
         self.destroy(gracefully=True, free_mac_addresses=False)
-        if not self.undefine():
+        undef_opts = "--managed-save --snapshots-metadata"
+        if libvirt_version.version_compare(7, 5, 0):
+            undef_opts += " --checkpoints-metadata"
+        if not self.undefine(options=undef_opts):
```

Comment:
I think it's not reasonable to set some 'fixed' opts to self.undefine() because of some migration cases' failures. I think it'll be great if you can fix this problem in migration_template.py only. What's your opinion? @dzhengfy

Reply:
The function here is "remove", not undefine.

Reply:
No, you're adding "--managed-save --snapshots-metadata" or "--managed-save --snapshots-metadata --checkpoints-metadata" to self.undefine() forcefully here.